packages/create-sitecore-jss/src/bin.ts

```diff
@@ -98,6 +113,23 @@ export const main = async (args: ParsedArgs) => {
     args.force = true;
   }
 
+  let preCommitHook = args.preCommitHook;
+  if (!preCommitHook) {
+    if (args.yes) {
+      preCommitHook = DEFAULT_PRECOMMITHOOK;
+    } else {
+      const answer = await inquirer.prompt({
+        type: 'list',
```

Comment:
I think we can use `type: 'confirm'` for a yes/no prompt, which then uses a boolean value. I think this might then resolve my previous 2 comments.